lib/promise.js

```diff
@@ -34,7 +24,6 @@ var SequelizePromise = function (resolver) {
 
   promise.seqResolve = this.seqResolve;
   promise.seqReject = this.seqReject;
```

Comment:
Not even sure we need to provide a SequelizePromise - modifying `Promise.prototype._then` with CLS support should be the only change neededd